### PR TITLE
MINI-2194 Settings saved values were not displayed

### PR DESCRIPTION
**Pull request by Climbatize**
# Description
When saving project id or subscription key, if you launched the settings screen again default value taken from config file were displayed instead of saved values.

## Links
MINI-2194

# Checklist
- [X] I have read the [contributing guidelines](CONTRIBUTING.md).
- [X] I have completed the [SDK Development Learning Path](CONTRIBUTING.md#SDK-Development-Learning-Path) (if submitting a feature PR)
- [ ] I wrote/updated tests for new/changed code
- [X] I removed all sensitive data before every commit, including API endpoints and keys
- [X] I ran `fastlane ci` without errors

### CODE REVIEW

File: Example/Controllers/SettingsTableViewController.swift [hunk 1]
```diff
@@ -172,11 +172,10 @@ class SettingsTableViewController: UITableViewController {
     }
 
     func getTextFieldValue(key: Config.Key, placeholderText: String?) -> String? {
-        if predefinedKeys.contains(placeholderText ?? "") {
-            return Config.userDefaults?.string(forKey: key.rawValue) ?? ""
-        } else {
+        guard let value = Config.userDefaults?.string(forKey: key.rawValue) else {
```
Comment (Climbatize) [inline Example/Controllers/SettingsTableViewController.swift:175]: this is where the issue was happening

File: Example/Controllers/SettingsTableViewController.swift [hunk 2]
```diff
@@ -10,7 +10,7 @@ class SettingsTableViewController: UITableViewController {
     @IBOutlet weak var invalidSubscriptionKeyLabel: UILabel!
     weak var configUpdateDelegate: SettingsDelegate?
 
-    let predefinedKeys: [String] = ["RAS_APPLICATION_IDENTIFIER", "RAS_SUBSCRIPTION_KEY", ""]
+    let predefinedKeys: [String] = ["RAS_PROJECT_IDENTIFIER", "RAS_SUBSCRIPTION_KEY", ""]
```
Comment (ts-leojoseph-r) [inline Example/Controllers/SettingsTableViewController.swift:13]: Sorry that I missed this.

**Review by ts-leojoseph-r — APPROVED** [review]
Looks good to me :+1:

**Review by corycaywood — APPROVED** [review]
LGTM 👍

**Pull request merged**